cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/toml/util/SingletonStack.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.launcher.toml.util;
+
+/**
+ * This class can be used to handle a singleton content with one element.
+ *
+ * @since 0.964
+ * @param <T>
```

Comment:
May be you could improve docs with description.

Reply:
Fixed with  ad5c1ff125c11049a9b1c7e8ecb185e8e1e5ed02